### PR TITLE
[task/view-model-unit tests]

### PR DESCRIPTION
**Pull request by shehzad-ismail8**

### CODE REVIEW

File: src/EPR.RegulatorService.Frontend.UnitTests/Web/ViewModels/RegistrationSubmissions/CancelDateRegistrationSubmissionViewModelTests.cs [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+                Day = 15,
+                Month = 10,
+                Year = 2023 // Past date
+            };
+
+            // Act
+            var validationResults = ValidationHelper.ValidateModel(viewModel);
+
+            // Assert
+            Assert.AreEqual(1, validationResults.Count, "Expected a validation error for a past cancellation date.");
```
Comment (prasadramireddyeviden) [inline src/EPR.RegulatorService.Frontend.UnitTests/Web/ViewModels/RegistrationSubmissions/CancelDateRegistrationSubmissionViewModelTests.cs:46]: To make code consistence,  would you please update your asserts with Fluent Assertions?

**Review by JulianYoungEviden — APPROVED** [review]

File: src/EPR.RegulatorService.Frontend.UnitTests/Web/ViewModels/RegistrationSubmissions/ConfirmOfflinePaymentSubmissionViewModelTests.cs [hunk 2]
```diff
@@ -0,0 +1,100 @@
+namespace EPR.RegulatorService.Frontend.UnitTests.Web.ViewModels.RegistrationSubmissions
+{
+    using EPR.RegulatorService.Frontend.UnitTests.Web.ViewModels.Shared.Helpers;
+    using EPR.RegulatorService.Frontend.Web.ViewModels.RegistrationSubmissions;
+
+    [TestClass]
+    public class ConfirmOfflinePaymentSubmissionViewModelTests
+    {
+        [TestMethod]
+        public void ConfirmOfflinePaymentSubmissionViewModel_ValidModel_ShouldHaveNoValidationErrors()
```
Comment (prasadramireddyeviden) [inline src/EPR.RegulatorService.Frontend.UnitTests/Web/ViewModels/RegistrationSubmissions/ConfirmOfflinePaymentSubmissionViewModelTests.cs:10]: Can you use 'DataRow' cover more scenarios in one TestMethod?

File: src/EPR.RegulatorService.Frontend.UnitTests/Web/ViewModels/RegistrationSubmissions/CancelDateRegistrationSubmissionViewModelTests.cs (hunk 1 above)
Comment (shehzad-ismail8) [reply src/EPR.RegulatorService.Frontend.UnitTests/Web/ViewModels/RegistrationSubmissions/CancelDateRegistrationSubmissionViewModelTests.cs]: Refactored

File: src/EPR.RegulatorService.Frontend.UnitTests/Web/ViewModels/RegistrationSubmissions/ConfirmOfflinePaymentSubmissionViewModelTests.cs (hunk 2 above)
Comment (shehzad-ismail8) [reply src/EPR.RegulatorService.Frontend.UnitTests/Web/ViewModels/RegistrationSubmissions/ConfirmOfflinePaymentSubmissionViewModelTests.cs]: Done

**Pull request merged**